tensorflow/core/lib/gtl/array_slice.h

```diff
@@ -191,8 +191,8 @@ class ArraySlice {
   void pop_front() { remove_prefix(1); }
 
   // These relational operators have the same semantics as the
-  // std::vector<T> relational operators: they do deep (elementwise)
-  // comparisons.  Array slices are equal iff their size is the same
```

Comment:
This is not a typo. "Iff" means "if and only if".
https://www.merriam-webster.com/dictionary/iff